pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.go

```diff
@@ -34,12 +34,21 @@ type Conf struct {
 	Tags *[]Tag `json:"tags,omitempty"`
 }
 
-// Only one of zipkin or datadog can be used.
+// Only one of zipkin, datadog or openTelemetry can be used.
 type Backend struct {
 	// Zipkin backend configuration.
 	Zipkin *ZipkinBackend `json:"zipkin,omitempty"`
 	// Datadog backend configuration.
 	Datadog *DatadogBackend `json:"datadog,omitempty"`
+	// OpenTelemetry backend configuration.
+	OpenTelemetry *OpenTelemetryBackend `json:"openTelemetry,omitempty"`
+}
+
+// OpenTelemetry tracing backend configuration.
+type OpenTelemetryBackend struct {
+	// Address of OpenTelemetry collector.
+	// +kubebuilder:example="otel-collector:4317"
+	Endpoint string `json:"endpoint"`
```

Comment:
```suggestion
    // +kubebuilder:validation:MinLength=1
	Endpoint string `json:"endpoint"`
```

Reply:
@michaelbeaumont - is there any reason it's not called `Url` to be consistent with other backends? ([zipkin](https://github.com/kumahq/kuma/blob/e84dc743f84ba21197bf1bcf4f01b973482ce4d2/pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.go#L58), [datadog](https://github.com/kumahq/kuma/blob/e84dc743f84ba21197bf1bcf4f01b973482ce4d2/pkg/plugins/policies/meshtrace/api/v1alpha1/meshtrace.go#L77))

Reply:
Would it be due to the absence of protocol in that field, which is defined for Zipkin and Datadog.